src/Controller/Model/ObjectTypesController.php

```diff
@@ -56,12 +102,58 @@ public function view($id): ?Response
 
         $objectTypeProperties = $this->prepareProperties((array)$response['data'], $name);
         $this->set(compact('objectTypeProperties'));
-        $this->set('schema', $this->Schema->getSchema());
+        $schema = $this->Schema->getSchema();
+        $schema['properties']['table'] = ['type' => 'string', 'enum' => $this->tables()];
+        $schema['properties']['parent_name'] = ['type' => 'string', 'enum' => $this->abstractTypes()];
+        $this->set('schema', $schema);
         $this->set('properties', $this->Properties->viewGroups($resource, $this->resourceType));
 
         return null;
     }
 
+    /**
+     * Get available tables list
+     *
+     * @return array
+     */
+    protected function tables(): array
+    {
+        $tables = array_unique(
+            array_merge(
+                self::TABLES,
+                (array)Configure::read('Model.objectTypesTables')
+            )
+        );
+        sort($tables);
+
+        return $tables;
+    }
+
+    /**
+     * Get abstract types from api
+     *
+     * @return array
+     */
+    protected function abstractTypes(): array
+    {
+        $response = [];
```

Comment:
Simpler version
```php
$features = $this->Schema->objectTypesFeatures();
$types = array_keys($features['descendants'));
sort($types);
```

we could also create a new method `SchemaComponent::abstractTypes()` with this logic